_features/model_predictive_control.md

```diff
@@ -59,6 +59,23 @@ MPC_AMBIENT_XFER_COEFF_FAN255 0.0998
 
 [`M306`](/docs/gcode/M306.html) can also be used to change constants at runtime.
 
+### Manual tuning
+
+If `M306 T` does not work, e.g. with positive temperature coefficient (PTC) hotends, MPC can be configured manually.
+
+1. Start with the part cooling fan off and hotend cold. Record the starting temperature Ts. e.g. Ts = 20&deg;C.
+1. Set the temperature to 200&deg;C and then back to 0&deg;C once it reaches around 200&deg;C. Measure the curve of temperature
+vs time whilst the hotend is heating, starting at time = 0s.
+1. Measure the fastest rate Rf at which temperature increases in &deg;C/s. e.g. Rf = 3&deg;C/s.
+1. Divide the heater power by Rf to get `MPC_BLOCK_HEAT_CAPACITY` in J/K. e.g. 40W / 3&deg;C/s = 13.33 J/K.
+1. Measure the temperature Tf and time tf of the point where temperature was increasing fastest.
+1. `MPC_SENSOR_RESPONSIVENESS` is Rf / (Rf x tf + Ts - Tf). e.g. for tf = 10s and Tf = 35&deg;C, this is 3&deg;C/s / ( 3&deg;C/s x 10s + 20&deg;C - 35&deg;C) = 0.2 K/s/K.
+1. Set these values with `M306` and set the temperature to 200&deg;C.
```

Comment:
That's unfortunate: `200&deg;C.` Are you sure markdown can't properly handle it?

Reply:
Once it is processed by jekyll, it comes out as a degree sign.